src/payment/strategies/cybersource/cybersource-payment-strategy.spec.ts

```diff
@@ -0,0 +1,216 @@
+import { createClient as createPaymentClient } from '@bigcommerce/bigpay-client';
+import { createAction, Action } from '@bigcommerce/data-store';
+import { createRequestSender } from '@bigcommerce/request-sender';
+import { createScriptLoader } from '@bigcommerce/script-loader';
+import { merge } from 'lodash';
+import { of, Observable } from 'rxjs';
+
+import {
+    createCheckoutStore,
+    CheckoutRequestSender,
+    CheckoutStore,
+    CheckoutValidator,
+} from '../../../checkout';
+import { OrderActionCreator, OrderActionType, OrderRequestBody, OrderRequestSender } from '../../../order';
```

Comment:
If you import more than 3 modules please separate in a new line.
💅
```suggestion
import {
  OrderActionCreator,
  OrderActionType,
  OrderRequestBody,
  OrderRequestSender
} from '../../../order';
```